src/views/studio/StudioNavButton/StudioNavButton.module.scss

```diff
@@ -0,0 +1,33 @@
+.button {
+    display: flex;
+    background-color: white;
+    margin-top: 15px;
```

Comment:
margin: 7px 0;
would give better result (this css is making this weird gap between studio and nav)